src/hairpin_creator.lua

```diff
@@ -25,7 +25,8 @@ function plugindef()
         hairpin_type = -1 -- "swell"
         hairpin_type = -2 -- "unswell"
     ]]
-    finaleplugin.MinJWLuaVersion = 0.62
+    finaleplugin.MinJWLuaVersion = 0.63
+    finaleplugin.ScriptGroupName = "Hairpin creator"
```

Comment:
I also added an option `finaleplugin.ScriptGroupDescription` which would then be the short description on the website instead of "Create crescendo spanning the selected region". Want to add it?

Other than that, LGTM.

Reply:
> Want to add it?

Yes please!